x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -561,23 +580,44 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
       );
     }
 
-    if (showViewSeriesLink === true && anomaly.isTimeSeriesViewRecord === true) {
-      items.push(
-        <EuiContextMenuItem
-          key="view_series"
-          icon="visLine"
-          onClick={() => {
-            closePopover();
-            viewSeries();
-          }}
-          data-test-subj="mlAnomaliesListRowActionViewSeriesButton"
-        >
-          <FormattedMessage
-            id="xpack.ml.anomaliesTable.linksMenu.viewSeriesLabel"
-            defaultMessage="View series"
-          />
-        </EuiContextMenuItem>
-      );
+    if (showViewSeriesLink === true) {
+      if (anomaly.isTimeSeriesViewRecord) {
+        items.push(
+          <EuiContextMenuItem
+            key="view_series"
+            icon="visLine"
+            onClick={() => {
+              closePopover();
+              viewSeries();
+            }}
+            data-test-subj="mlAnomaliesListRowActionViewSeriesButton"
+          >
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.linksMenu.viewSeriesLabel"
+              defaultMessage="View series"
+            />
+          </EuiContextMenuItem>
+        );
+      }
+
+      if (anomaly.isGeoRecord === true) {
+        items.push(
+          <EuiContextMenuItem
+            key="view_in_maps"
+            icon="logoMaps"
+            onClick={async () => {
+              const mapsLink = await getMapsLink(anomaly);
+              await application.navigateToApp(MAPS_APP_ID, { path: mapsLink?.path });
+            }}
+            data-test-subj="mlAnomaliesListRowActionViewInMapsButton"
+          >
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.linksMenu.viewInMapsLabel"
+              defaultMessage="View in maps"
```

Comment:
Should use capitalized `Maps` here I think.

Reply:
Updated in 1a4013bf62e01b1d3f5cde8fd6454843de4ccb2e